docs/blog/author.yaml

```diff
@@ -296,3 +296,7 @@
   bio: Software developer @ GatsbyJS. Likes solving weird problems, skiing and watching/reading sci-fi. In that order.
   avatar: avatars/michal-piechowiak.jpeg
   twitter: "@mipiechowiak"
+  -id: Michelle Gienow
+  bio: Writes things @ GatsbyJS. Chaotic good frontend dev, caffeine addict, backyard chicken wrangler 🐓. 
+  avatar: avatars/michelle-gienow.jpg
```

Comment:
This needs to exactly match the name of the avatar file!

So need to rename the JPG to ipg